docs/classic/connect_four.md

```diff
@@ -15,15 +15,15 @@ agent-labels: "agents= ['player_0', 'player_0']"
 
 
 
-Connect Four is a 2 player turn based game, where players must connect four of their tokens vertically, horizontally or diagonally. The players drop their respective token in a standing grid, where each token will fall as far down into its column as possible. Players cannot fill up a column with more than 6 token, and the game ends when either a player has made a sequence of 4 token, or when all 7 columns have been filled.
+Connect Four is a 2-player turn based game, where players must connect four of their tokens vertically, horizontally or diagonally. The players drop their respective token in a column of a standing grid, where each token will fall until it reaches the bottom of the column or another token. Players cannot place a token in a full column, and the game ends when either a player has made a sequence of 4 tokens, or when all 7 columns have been filled.
```

Comment:
"an existing token"?

unnecessary either in last sentence?

Reply:
do you want me to replace "another token" with "an existing token"?
I can remove "either"
Also I should add a comma after horizontally

Reply:
"do you want me to replace "another token" with "an existing token"?" I mean that seems better to me at least?

Reply:
Yeah I agree. I just wasnt sure what your comment was referring to. I'll change it